sync/sync_test.go

```diff
@@ -350,7 +350,7 @@ func TestSync_ShouldRegisterMissingRegistrationInMultiregistrationScenario(t *te
 }
 
 /*
-This may happen if an application configuration is changed and then the deployment this triggered is rolled back.
+This may happen if an application configuration is changed and then the deployment this triggered is rolled back (deployment stopped).
```

Comment:
There's no such thing as **stopping deployment** in current releases of Marathon. You can `cancel`/`delete` deployment **with** or **without** a new **rollback deployment** https://github.com/mesosphere/marathon/blob/master/docs/docs/rest-api/mesosphere/marathon/api/v2/DeploymentsResource_cancel.md